File: assignments/solutions/02-first-java/currencyConversion.js
/**
 * Currency Conversion (INR to USD)
 * 
 * @param {number} input
 * @returns {number}
 */

const readline = require('readline');
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

const currencyConversion = (input) => {
    return input / 75;
}

console.log('Currency Conversion (INR to USD)');

rl.question('Enter Amount (in INR): ', input => {
    console.log(`₹${input} = $${currencyConversion(input).toFixed(3)}`)
    rl.close();
});

/**
 * Output:
 *
 * Currency Conversion (INR to USD)
 * Enter Amount (in INR): 200
 * ₹200 = $2.667
 */

File: assignments/solutions/02-first-java/currencyConversion.java
// Currency Conversion (INR to USD)

import java.util.Scanner;

public class currencyConversion {
    public static void main (String args[]) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Currency Conversion (INR to USD)");
        System.out.print("Enter Amount (in INR): ");
        float input = scan.nextFloat();
        double result = input / 75;
        System.out.printf("₹%.2f = $%.3f\n", input, result);
    }
}

/*
Output:

Currency Conversion (INR to USD)
Enter Amount (in INR): 200
₹200.00 = $2.667
*/